pkg/apis/options/legacy_options.go

```diff
@@ -579,6 +580,7 @@ func legacyProviderFlagSet() *pflag.FlagSet {
 	flagSet.String("oidc-email-claim", OIDCEmailClaim, "which OIDC claim contains the user's email")
 	flagSet.StringSlice("oidc-audience-claim", OIDCAudienceClaims, "which OIDC claims are used as audience to verify against client id")
 	flagSet.StringSlice("oidc-extra-audience", []string{}, "additional audiences allowed to pass audience verification")
+	flagSet.StringSlice("oidc-public-key-file", []string{}, "path to public key file in PEM format to use for verifying JWT tokens (may be given multiple times)")
```

Comment:
Needs to be added to the overall docs page:
https://github.com/tuunit/oauth2-proxy/blob/feature/static-public-keys-support/docs/docs/configuration/overview.md